focuspoints.lrdevplugin/focus_points/pentax/pentax k-50.txt

```diff
@@ -2,8 +2,6 @@
 -- Submitted by: roguephysicist
 -- 16.3 MP, 4928x3264
 -- AF System: 11 points, 9 cross-type
--- selected points: AFPointSelected
--- in-focus points: AFPointsInFocus
```

Comment:
Suggestion: The "result" you return can take an width/height of the focus point. I saw that your focus points on the pentax k-50 have a tall left-side-point and tall-right-side-point. It would be cool to perhaps add that so it's more on point. I'm not sure right now how to abstract that out for each camera. Perhaps another node in the file. Perhaps the point nodes themself can have the focus point dimen like this

```
-- center row
Left        = {1045, 1632},{250, 450}
Mid-left    = {1740, 1632}
Center      = {2464, 1632}
Mid-right   = {3188, 1632}
Right       = {3883, 1632},{250, 450}
```